### DISCUSSION TITLE
Thanks for the time

### DISCUSSION THREAD

**Discussion by GentleJack72** [practical-tutorials/project-based-learning#401] [General]
I'm new here and knows nothing about programming but will love to learn how do i start

**Comment by calvinkeyth** [comment]
Learning is the key and training class soon

**Comment by GentleJack72** [comment]
How do I learn

**Reply by ghost** [reply]
What type of programming or job are you interested in? If you are interested in data science, you might want to start with introductory Python or R-based classes. For web development, focus on learning HTML, CSS, JavaScript, and React.

**Comment by shindodkar** [comment]
would anyone one suggest me projects on web development beginner friendly.

**Reply by weshaan** [reply]
stone papers scissors website

**Reply by microwavedd** [reply]
A drum kit website using javascript to add sounds

**Reply by M0hanrajp** [reply]
> would anyone one suggest me projects on web development beginner friendly.

You could try cloning Apple's webpage ( try to create a web page for a single product ).\
Keeping in mind that the web page needs to be interactive and aesthetic.

**Reply by ghost** [reply]
A to-do list that follows you while you browse (first create a basic to-do list app that works appropriately standalone. Later on, convert the web app to a browser extension using javascript or other APIs to test yourself).

**Comment by archmikey** [comment]
new to projects? start with web development (web3) learn basics like centralized and decentralized finances. what is a private key? what is a public key? which chains operate under the ETC20 standard and what are the implications under circumstances. which networks and blockchains are interoperable and how stable are the bridges?
i've developed with several operating blockchains including PulseChain and not yet released X1
